index.bs

```diff
@@ -168,6 +168,12 @@ type: dfn
       which was initiated by a secure context but is downloaded over an
       insecure connection.
     </dd>
+
+    <dt><dfn export>non-loopback IP address</dfn></dt>
+    <dd>
+      A non-loopback IP address is an <a>IP address</a> that is not contained in
+      CIDR `127.0.0.0/8` or `::1/128` [[!RFC4632]].
```

Comment:
Typically references go after the dot though I see this document uses a different style...